### PR TITLE
📝 Clarify that we use a branch and merge strategy

### PR DESCRIPTION
**Pull request by zspencer**
Technically we don't prevent maintainers from writing to release
branches, but it seems useful to call let contributors know that we
prefer that they use branches.

### CODE REVIEW

**Comment by drllau** [comment]
Then in the core/HOWTO.md put up some general guidelines (like?)
1. anything in /core only branches assigned to Zinc Coop members or people approved by Zee
2. create /contribute/[yourname] for optional stuff that can be switched on/off at high level
3. /common is public domain stuff like images, or stuff which anyone can reference

Then have some common branches that people are working on and where to put patches or credit. Note 2 is risk management in that sometimes you need time to vet origins before merging into core. 

PS I'm still trying to figure out how to upgrade to latest master ... github not exactly obvious in some places. In the end just deleted forked repo to restart.

**Comment by zspencer** [comment]
I don't want to set up a `common` or `contrib` folders just yet for two reasons:

1. It would require people to have to know yet another thing (where to put things based upon their goals) instead of pushing everything as core.
2.  The Ruby programming language is designed to be extended. Anything that people would want to put in `contrib` can be written and packaged independently and installed next-to compensated. 

I used core/common/contrib back in the old SVN/CVS/SourceSafe days, back when only one person could "checkout" a copy of a file at the same time. 

Git and Mercurial side-step that quite nicely with their repository + branch pattern; where it's entirely possible for someone to have a downstream version of their own that they pull in upstream changes. The git-scm book has a great section on [branching workflows.](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows) and [distributed workflows](https://git-scm.com/book/en/v2/Distributed-Git-Distributed-Workflows#ch05-distributed-git).

We're currently doing an "Integration Manager Distributed Workflow" with the folks in the @zinc-collective/compensated-maintainers group acting as Integration Managers.

**Comment by drllau** [comment]
Came across this which [automates contribution agreements ](https://github.com/clahub/clahub)(somewhat).

**Comment by user512** [comment]
Merging because I see this PR fits its description.

**Review by user512 — APPROVED** [review]

**Pull request merged**
